pyproject.toml

```
@@ -75,7 +75,7 @@ optional-dependencies.extra = [
 optional-dependencies.test = [
   "einops>=0.7",
   "protobuf>=4.23.4",
-  "pydantic>=2.11",
+  "pydantic",
```

```suggestion
  "pydantic>=2.11",
```
why this is needed? 🤔

---

let me revet it just to see the CI output...